packages/astro/test/fixtures/content-collections/.gitignore

```diff
@@ -1 +1 @@
-types.generated.d.ts
+.astro
```

Comment:
This could probably go in the root `.gitignore`?

Reply:
Agreed! I'll move